src/prefs/PrefsPanel.h

```diff
@@ -102,7 +102,7 @@ class AUDACITY_DLL_API PrefsPanel /* not final */
    virtual bool Commit() = 0; // used to be called "Apply"
 
 
-   virtual PluginPath GetPath();
```

Comment:
“override” is missing from these function declarations.  It would be nice to add a commit that supplies it.  Not essential though for this issue.